charts/pega/README.md

```diff
@@ -762,6 +762,21 @@ dds:
   username: "dnode_ext"
   password: "dnode_ext"
 ```
+### Deploying Pega without Cassandra
+You may configure Pega without any Cassandra datastore. When deploying in this mode set `cassandra.enabled` to `false` and comment out or delete the `dds` section.
```

Comment:
how about match the statement to the heading>>
To configure a Pega Platform deployment without a Cassandra datastore (DDS), set `cassandra.enabled` to `false` and comment out or delete the `dds` section.